util.go

```diff
@@ -33,3 +35,20 @@ func (v VersionFlag) BeforeApply(app *Kong, vars Vars) error {
 	app.Exit(0)
 	return nil
 }
+
+// ChangeToFlag changes the current working directory to a path specified by a flag
+// early in the parsing process, changing how other flags resolve relative paths.
+//
+// Use this flag to provide a "git -C" like functionality.
+type ChangeToFlag string
```

Comment:
Can you rename this to `ChangeDirFlag` so it's a bit clearer?

Reply:
Sure.